src/uct/cuda/cuda_copy/cuda_copy_ep.c

```diff
@@ -158,6 +158,9 @@ uct_cuda_copy_post_cuda_async_copy(uct_ep_h tl_ep, void *dst, void *src,
     ucs_queue_push(event_q, &cuda_event->queue);
     cuda_event->comp = comp;
 
+    UCS_BITMAP_SET(iface->streams_to_sync,
+                   src_type * UCS_MEMORY_TYPE_LAST + dst_type);
+
```

Comment:
is uct_cuda_copy_post_cuda_async_copy being called in the case of ucx_perftest hang? (when copying sn from host to device)

Reply:
No, but the short requests already calling cuStreamSync